### PR TITLE
Inflight Magazine no. 5

### PR DESCRIPTION
**Pull request by dannac2**
# Docs updates should not be submitted in this repository

Our doc site content is being automatically updated from [Wing](https://github.com/winglang/wing) repository docs folder.

Please submit any changes to the docs as a PR [here](https://github.com/winglang/wing/pulls)

### CODE REVIEW

**Comment by dannac2** [comment]
@MarkMcCulloh can you please review this, it's a summary of the changelog. Want to make sure everything is correct. Also I have no idea what i'm doing wrong with the images, can you tell me what i'm doing wrong?

**Comment by dannac2** [comment]
@MarkMcCulloh making sure you saw my comment above, thanks!

**Comment by dannac2** [comment]
@eladb ready for your review!!

File: blog/2023-09-20-magazine-005.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  - dcahana2
+tags: [cloud-oriented programming, winglang, changelog, newsletter,]
+hide_table_of_contents: true
+---
+> The 5th issue of the Wing Inflight Magazine.
+> <!--truncate-->
+
+Hey folks!
+
+We’re back with another overview of our CHANGELOG - a new segment in our quirky Twitch show called [The Wingly Update](https://www.youtube.com/playlist?list=PL-P8v-FRassZBWsNoSafL_ReO0JO0xJVm) where we share news about new features to Wing, so we decided to share the same updates here as well. Let us know what you think!
```
Comment (eladb) [inline blog/2023-09-20-magazine-005.md:14]:
```suggestion
We’re back with another issue of the Wing Inflight Magazine and excited to share some updates from our project.
```

File: blog/2023-09-20-magazine-005.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+- 🚀 [New: Added set API to MutArray](#added-set-api-to-mutarray)
+- 🔥 [Runtime checked Json => Struct conversion](#runtime-checked-json)
+- 🔥 [Json implicit struct casting](#json-implicit-struct-casting)
+- 🚀 [New: Added addFile API to Bucket](#added-addfile-api-to-bucket)
+- 🚀 [New: Wing Libraries](#wing-libraries)
+- 🔥 [Updated Directory Modules](#directory-modules)
+- 🚀 [New: fs Namespace](#fs-namespace)
+- 🔥 [Breaking Changes](#additional-breaking-changes)
+- 🔥 [New Features](#additional-new-features)
+- 📅 [Upcoming Events](#upcoming-events)
```
Comment (eladb) [inline blog/2023-09-20-magazine-005.md:26]: Can we add a section with links to all of the Wingly shorts we created so far?

**Review by eladb — CHANGES_REQUESTED** [review]

File: blog/2023-09-20-magazine-005.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+- 🚀 [New: Added addFile API to Bucket](#added-addfile-api-to-bucket)
+- 🔥 [Breaking Changes](#breaking-changes)
+- 🔥 [New Features](#new-features)
+- 📅 [Upcoming Events](#upcoming-events)
+
+
+### Variadic Arguments
+❤️ Added by [Marcio Cruz](https://github.com/marciocadev) & [Chris Rybicki](https://github.com/Chriscbr)  ❤️
+
+Winglang now supports Variadic Arguments, both for declaring and consuming within the language itself. This feature is also extended to support JSII libraries, filling a gap that might have been a blocker for some users. Essentially, the addition provides full support for JSII libraries, expanding its usability.
```
[the remark this answers is not in the document]
Comment (dannac2) [reply blog/2023-09-20-magazine-005.md]: not sure how to really fix it honestly, @MarkMcCulloh can you possibly help?

File: blog/2023-09-20-magazine-005.md (hunk 2 above)
Comment (dannac2) [reply blog/2023-09-20-magazine-005.md:26]: > Can we add a section with links to all of the Wingly shorts we created so far?

done @eladb

File: blog/assets/2023-09-20-magazine-005/json_casting_2.png
Comment (eladb) [inline blog/assets/2023-09-20-magazine-005/json_casting_2.png:1]: seems like the same as `json_casting_1` no?

File: blog/assets/2023-09-20-magazine-005/mutarray_2.png
Comment (eladb) [inline blog/assets/2023-09-20-magazine-005/mutarray_2.png:1]: seems like a duplicate

File: blog/2023-09-20-magazine-005.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
+---
+title: Inflight Magazine no. 5
+description: Updates from Winglang
+authors: 
+  - dcahana2
```
Comment (eladb) [inline blog/2023-09-20-magazine-005.md:5]: maybe add mark?

File: blog/2023-09-20-magazine-005.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+**🔥 Live Workshop - December 6th @4:30 - 6pm IST**
+Get hands-on with Eyal as he walks us through the process of constructing a full-fledged React.js application with Winglang.
+For more information and to register, [click here](https://www.eventbrite.com/e/winglang-react-workshop-tickets-754616256537?aff=oddtdtcreator).
+
+### The Wingly Update
+Have you heard about **[The Wingly Update](https://www.twitch.tv/winglangio)**? It's our biweekly show where we stream to Twitch, share the latest developments of the project, and just make a bunch of fools of ourselves. Mark your calendars and make sure to catch us live!
+
+If you haven't been able to catch our show, don't fret! You can find the complete stack of [all our episodes here](https://youtube.com/playlist?list=PL-P8v-FRassZBWsNoSafL_ReO0JO0xJVm&si=trffVrtGGMUZ-SKb). 
+
+You can check out some cool highlights from our show below!
```
Comment (eladb) [inline blog/2023-09-20-magazine-005.md:100]: let's add more segments here. maybe ~10? maybe just focus on interviews?
also - i wouldn't put the changelog segments, just interviews.

**Review by eladb — CHANGES_REQUESTED** [review]

File: blog/assets/2023-09-20-magazine-005/json_casting_2.png
Comment (ShaiBer) [reply blog/assets/2023-09-20-magazine-005/json_casting_2.png:1]: No, one is the before and the other is the after

File: blog/assets/2023-09-20-magazine-005/mutarray_2.png
Comment (ShaiBer) [reply blog/assets/2023-09-20-magazine-005/mutarray_2.png:1]: one is set and the other is insert

File: blog/2023-09-20-magazine-005.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+hide_table_of_contents: true
+---
+> The 5th issue of the Wing Inflight Magazine.
+> <!--truncate-->
+
+Hey folks!
+
+We’re back with another issue of the **Wing Inflight Magazine** and we are excited to share some updates about [winglang](https://winglang.io).
+
+- 🚀 [New: Variadic Arguments](#variadic-arguments)
```
[the remark this answers is not in the document]
Comment (ShaiBer) [reply blog/2023-09-20-magazine-005.md:17]: I committed a fix to a few links now

**Review by ShaiBer — APPROVED** [review]

**Pull request merged**